packages/destination-actions/src/destinations/heap/trackEvent/index.ts

```diff
@@ -11,7 +11,7 @@ type HeapEvent = {
   event: string | undefined
   idempotency_key: string
   timestamp?: string
-  properties: {
+  properties?: {
```

Comment:
I'm concerned about making changes to the mappings. Also, I think properties should be required. Would the event be useful without them?

Reply:
oh it was throwing in the PR, should i just make an empty properties object?

Reply:
my payload in testing didnt have properties object